README.md

```diff
@@ -21,30 +16,24 @@ This datapack adds the ability to find and detect slime chunks in game without o
 
 ## Setup
 
-Before you begin, you must enter your seed into the file located at `data/slime_chunk_finder/functions/config/seed.mcfunction`. Because this datapack employs scoreboards in order to run, you must first convert your seed into two 32 bit integers. (A program to do so can be found within config/seed in the functions folder. Click on the HTML file to run it in your browser.)
+Before you begin, you must enter your seed into the file located at `data/slime_chunk_finder/functions/config/seed.mcfunction`. Because this datapack employs scoreboards in order to run, you must first convert your seed into two 32 bit integers. (A web program to do so can be found within `data/slime_chunk_finder/functions/config`. Click on the HTML file to run it in your browser.)
 
-## Crafting the Slime Finder
+## Obtaining the Slime Chunk Finder Item
 
-A Slime Finder item can be crafted by throwing a Slimeball on the ground and then throwing a Compass on top of it. They will "craft" together into a Slime Finder item.
+A Slime Chunk Finder item can be found in about 25% of dungeon loot chests and about 50% of jungle temple loot chests.
```

Comment:
What are your thoughts on instead having this be a rare (1%?) drop from Slimes? - Have to hunt a few before getting one.

Reply:
I thought about that. I feel like it's more logical to put them in loot chests (why would slimes drop a magical redstone item?), though I'm not sure how common they should be in order to be balanced.